.travis/script.sh

```diff
@@ -29,16 +46,25 @@ show_logs_and_return_non_zero() {
     cat ~/reserved_worker-1.log
     return "${rc}"
 }
+
+# Start services
+gunicorn pulpcore.tests.functional.content_with_coverage:server --bind 'localhost:8080' --worker-class 'aiohttp.GunicornWebWorker' -w 2 >> ~/content_app.log 2>&1 &
+CONTENT_APP_PID=$!
+coverage run $(which pulp-manager) runserver --noreload >> ~/django_runserver.log 2>&1 &
+PULP_SERVER_PID=$!
+echo "CONTENT_APP_PID=$CONTENT_APP_PID"
+echo "PULP_SERVER_PID=$PULP_SERVER_PID"
+
+wait_for_pulp 20
 pytest -v -r sx --color=yes --pyargs pulpcore.tests.functional || show_logs_and_return_non_zero
 pytest -v -r sx --color=yes --pyargs pulp_file.tests.functional || show_logs_and_return_non_zero
-codecov
 
-# Travis' scripts use unbound variables. This is problematic, because the
```

Comment:
I think, this is only a problem because we used `source` to call the scripts. Is there any (still valid) reason to do so?

Reply:
I don't know of a reason.